### PR TITLE
pyhecdss recipe submittal

### PR DESCRIPTION
**Pull request by dwr-psandhu**
Checklist

- [ x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x ] Source is from official source
- [x ] Package does not vend other packages
- [x ] Build number is 0
- [x ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/pyhecdss/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - python
+    - setuptools
+    - numpy {{ numpy }}
+    - swig
+    - pandas >=0.23
```
Comment (scopatz) [inline recipes/pyhecdss/meta.yaml]: Most of these that are not the compilers should be `host` deps

File: recipes/pyhecdss/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  #url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://github.com/CADWRDeltaModeling/pyhecdss/tarball/{{version}}
+
+build:
+  number: 0
+  #skip:           true
+  skip: true # [osx]
+  skip: true # [py<35]
+  skip: true # [win and vc<14]
+  script: LDSHARED=g++ python setup.py install --single-version-externally-managed --record=record.txt # [unix]
```
Comment (scopatz) [inline recipes/pyhecdss/meta.yaml]: If you need gcc, please depend on GCC, rather than compilers("cxx")

File: recipes/pyhecdss/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  requires:
+    - pytest
+    - pytest-cov
+  commands:
+    - (setlocal && cd tests && pytest && endlocal) # [win]
+    - (cd tests; pytest) # [unix]
+about:
+  home:           "https://github.com/CADWRDeltaModeling/pyhecdss.git"
+  license:        "MIT"
+  license_family: "MIT"
```
Comment (scopatz) [inline recipes/pyhecdss/meta.yaml]: This needs a `license_file` entry

File: recipes/pyhecdss/meta.yaml (hunk 2 above)
Comment (dwr-psandhu) [reply recipes/pyhecdss/meta.yaml]: Can i list g++ as the dependency instead. See my build line for unix

File: recipes/pyhecdss/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  summary:        "Read HECDSS timeseries as pandas DataFrame"
```
Comment (scopatz) [inline recipes/pyhecdss/meta.yaml]: There needs to be an `extras` section with the maintainers. Please see the example

File: recipes/pyhecdss/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,56 @@
+{% set name = "pyhecdss" %}
+{% set version = "0.2.4" %}
+
+package:
+  name:           "{{ name|lower }}"
+  version:        "{{ version }}"
+
+source:
+  #url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```
Comment (scopatz) [inline recipes/pyhecdss/meta.yaml]: please remove dead code

File: recipes/pyhecdss/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+  url: https://github.com/CADWRDeltaModeling/pyhecdss/tarball/{{version}}
+  sha256: 0821b24ca5fa6e2635495badfb00aa73f61d5abc4355def87e6e1184b69d5e8c
+
+build:
+  number: 0
+  #skip:           true
+  skip: true  # [osx]
+  skip: true  # [py<35]
+  skip: true  # [win and vc<14]
+  script: LDSHARED=g++ python setup.py install --single-version-externally-managed --record=record.txt  # [unix]
```
Comment (scopatz) [inline recipes/pyhecdss/meta.yaml]: Also, you shouldn't need LDSHARED anymore

**Comment by dwr-psandhu** [comment]
Looks like some internal error? Where do we go from here?

**Comment by scopatz** [comment]
Restarting the build

**Comment by dwr-psandhu** [comment]
win64 recipe fails with cryptic error. Any chance to see more information ?
[See this link]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=70650&view=logs&jobId=240f1fee-52bc-5498-a14a-8361bde76ba0&taskId=23010c01-6a9d-5f94-9037-792618644b77&lineStart=705&lineEnd=707&colStart=1&colEnd=50 )

**Comment by dwr-psandhu** [comment]
Similar issue on linux. Not enough information to debug 
[See this link](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=70650&view=logs&jobId=10bda42a-d420-5c28-a43f-bc4e66405e5c&taskId=bef775d0-a1a2-55a3-9721-17147029baf0&lineStart=458&lineEnd=478&colStart=1&colEnd=39)

File: recipes/pyhecdss/meta.yaml (hunk 2 above)
Comment (scopatz) [reply recipes/pyhecdss/meta.yaml]: That is generally not advised.

**Comment by dwr-psandhu** [comment]
@scopatz How can i trigger a build ? I updated the meta.yaml file but have not been able to see a build

**Pull request closed without merging**